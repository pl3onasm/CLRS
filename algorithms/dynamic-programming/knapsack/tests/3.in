100
(10,50.30),(15,30.50),(20,55.00),(5,10.50),(35,60.45),(41,35.60),(25,72.00),(2,10.75),(17,12.35),(50,45.40),(18,49.90),(35,36.50),(27,26.45),(23,55.00),(33,40.10)