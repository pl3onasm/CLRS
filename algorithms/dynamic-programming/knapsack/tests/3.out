Items included:
  (23,55.00)
  (18,49.90)
  (2,10.75)
  (25,72.00)
  (20,55.00)
  (10,50.30)

Total value: â‚¬ 292.95
Total weight: 98 kg
